### PR TITLE
Add android note

### PR DESCRIPTION
**Pull request by jbolinger**
It took awhile to get the examples reviewed, but they came though today. There are three in total but I just linked to one of the smaller ones and the example repo to keep it short.

\cc @timburks and @neozwu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,11 +221,16 @@ threading restrictions.
 
 Thus, the following are not supported:
 
-- Android
+- Android (consider [Firebase](https://firebase.google.com), which includes many of these APIs)
 - Alpine Linux (due to netty-tcnative requiring glibc, which is not present on Alpine)
 - Raspberry Pi (since it runs on the ARM architecture)
 - Google App Engine Standard Java 7
 
+*Android users*: Although unsupported, it is possible to use these libraries in many cases.
```
Comment (garrettjonesgoogle) [inline README.md]: I think I'd prefer this to be a subbullet of the Android bullet above

File: README.md [hunk 2]
```diff
@@ -221,11 +221,16 @@ threading restrictions.
...
+You can find examples, such as [this one](https://github.com/GoogleCloudPlatform/android-docs-samples/tree/master/speech/SpeechRecognitionClient),
+[here](https://github.com/GoogleCloudPlatform/android-docs-samples) but consider the risks carefully before
```
Comment (garrettjonesgoogle) [inline README.md]: the "here" text reads oddly here

Comment (jbolinger) [reply README.md]: how's `example repository` instead?

File: README.md (hunk 1 above)
Comment (jbolinger) [reply README.md]: sure, I moved it.

**Comment by jbolinger** [comment]
@garrettjonesgoogle I forgot about this one... how's this look?

**Pull request merged**